tests/system/action/personal_note/test_create.py

```diff
@@ -43,18 +44,13 @@ def test_create_no_star_and_no_html(self) -> None:
         )
 
     def test_create_not_unique(self) -> None:
-        self.set_models(
-            {
-                "meeting/110": {"name": "name_meeting_110"},
-                "motion/23": {"meeting_id": 110},
-                "personal_note/1": {
-                    "star": True,
-                    "note": "blablabla",
-                    "user_id": 1,
-                    "content_object_id": "motion/23",
-                },
-            }
-        )
+        self.test_model["personal_note/1"] = {
+            "star": True,
+            "note": "blablabla",
+            "user_id": 1,
+            "content_object_id": "motion/23",
+        }
+        self.set_models(self.test_model)
```

Comment:
```suggestion
        self.set_models({
            **self.test_model,
            "personal_note/1": {
                "star": True,
                "note": "blablabla",
                "user_id": 1,
                "content_object_id": "motion/23",
            }
        })
```
I think this would be more clear, else it seems like you overwrite something in the test models

Reply:
Okay, there are more locations of this idiom. Changed them in the tests.

Reply:
I'll try to use a double `set_models` instead of patching `test_models`. Perhaps that works.

Reply:
But `personal_note/1` is not in the test models? If it is, you previous approach is ok, because then you actually overwrite it, whihc is also done in other places.

Reply:
Ah, you just mean this special case, where the `personal_note/1` is not in the model.

Reply:
Yes, but it's not that important. You can leave it as-is if it works.